src/calculateRentalCost.js

```diff
@@ -3,8 +3,23 @@
  *
  * @return {number}
  */
+
+const daysWithLowADiscount = 3;
+const daysWithLargeADiscount = 7;
+const rentOfEveryDay = 40;
+const lowDiscount = 20;
+const largeDiscount = 50;
+
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= daysWithLowADiscount && days < daysWithLargeADiscount) {
+    return days * rentOfEveryDay - lowDiscount;
```

Comment:
The check for the lower discount range is correct, but it's not necessary to check if `days` is less than `daysWithLargeADiscount` because the subsequent condition handles the case when `days` is greater than or equal to `daysWithLargeADiscount`. You can remove the upper bound check in this `if` statement.